.github/workflows/create-rc-tag.yml

```diff
@@ -0,0 +1,97 @@
+name: Create RC Tag
+# Required secrets for this workflow: MAINTAIN_USERNAME, MAINTAIN_EMAIL, SLACK_TOKEN, MAINTAIN_TOKEN, and a var: SLACK_TAG_CHANNELS.
```

Comment:
```suggestion
# Required secrets for this workflow: SLACK_TOKEN and a var: SLACK_TAG_CHANNELS.
```

Reply:
Remove except `MAINTAIN_TOKEN` this property is used by `Generate changelog` & `softprops/action-gh-release`